libs/langchain/ragstack_langchain/graph_store/extractors/html_link_edge_extractor.py

```diff
@@ -1,6 +1,6 @@
 from langchain_core.documents import Document
-from ragstack_knowledge_store.link_tag import (
-    get_link_tags,
+from ragstack_langchain.graph_store.links import (
```

Comment:
I think it may be worth putting the edge extractor interface in the common code so we could share it with LlamaIndex, etc. That may be enabled by the EdgeExtractor change.

Reply:
It's always better to make it more generic.
Howver if we want to upstream extractors, I think we shouldn't have a dependency on  ragstack for them.
So LangChain and Llama-index would have their own copy/implementation of them.
WDYT ?

Reply:
Let's see that as a follow-up